src/rpc/misc.cpp

```diff
@@ -345,7 +345,7 @@ static UniValue setmocktime(const JSONRPCRequest& request)
                 RPCExamples{""},
             }.Check(request);
 
-    if (!Params().MineBlocksOnDemand())
+    if (!Params().IsMockableChain())
         throw std::runtime_error("setmocktime for regression testing (-regtest mode) only");
```

Comment:
Non-blocking nit: if you retouch this to add the opening and closing curly braces, could you also add the missing "is" in the error message? Like in https://github.com/bitcoin/bitcoin/pull/18037/files#diff-ad6efdc354b57bd1fa29fc3abb6e2872R383. Happy to re-ACK.

Reply:
done [e8600d9](https://github.com/bitcoin/bitcoin/pull/18263/commits/e8600d9f2d2c1b38257d91a3cf7eba670b139465)